README.md

```diff
@@ -49,6 +54,7 @@ POSTGRES_USER=atlasprd3 \
 POSTGRES_PASSWORD=atlasprd3 \
 docker-compose up
 ```
+Note: Update DB details like POSTGRES_DB, POSTGRES_USER, and POSTGRES_PASSWORD if you want to use different details, and you need to change DB details here as well `profile-docker. gradle` file if you go with different DB details.
```

Comment:
Too many “details” maybe? :smile:

How about this:
You can change the environment variables `POSTGRES_DB`, `POSTGRES_USER` and `POSTGRES_PASSWORD` to better suit your taste and environment; however remember to also adjust your `build-profiles/profile-docker.gradle` settings as well.